eg/Classes/ActionThread.py

```diff
@@ -102,6 +103,20 @@ def StartSession(self, filename):
                 print "   -", pluginInfo.name
             print "If you want to use them, please add the missing plugins."
 
+        payload = dict(
+            IsLocalAdmin=User.IsLocalAdmin(),
+            IsDomainLogin=User.IsDomainLogin()
+        )
+
+        if payload['IsDomainLogin']:
+            payload['IsDomainAdmin'] = User.IsDomainAdmin()
+
+        event = eg.EventGhostEvent(
+            prefix='EventGhost',
+            suffix='LoggedInUser.' + User.Name(),
+            payload=payload
+        )
+        event.Execute()
```

Comment:
Why are you not directly calling `eg.TriggerEvent`?

Reply:
This is a complex issue. But in short I did not want EG to continue loading until after whatever it is that needs to be changed by this event gets done. as an example. In a remote desktop environment some plugins like the Audioendpoint plugin do not function and cause EG to hang. so if you set up a specific username that you will use to login via RDP when EG starts this event gets triggered indicating the username so it would disable the plugin before it loads. and because the autostart macro gets processed differently and doesn't get added to the action thread queue the load of EG needs to stop loading take care of the tasks associated with this event and then continue. at this point if audio endpoint has been disabled then the autostart macro will not try to load the plugin.